app/controllers/admin/payments_controller.rb

```diff
@@ -33,8 +27,7 @@ def index
 
     # Get members of which the activities have been mailed 4 times, but haven't paid yet
     @late_activities = Activity.debtors.select do |activity|
-      activity.impressionist_count(message: "mail",
-                                   start_date: activity.start) >= 4 && activity.is_payable
+      (Date.today.prev_occurring(:friday) - activity.is_payable_updated_at.to_date).to_i >= 21 && activity.is_payable
```

Comment:
maybe add a comment here explaining that this counts up to 4 weeks (last friday is more then 21 days ago)

Reply:
Can you also remove the superfluous comment on line 28?